### PR TITLE
Update react-native to 0.60.5

### PR DESCRIPTION
**Pull request by friederbluemle**
This updates the `react-native` dependency to `0.60.5`, as well as all `devDependencies` according to: https://react-native-community.github.io/upgrade-helper/?from=0.59.4&to=0.60.5

Since React Native 0.60+ now includes a **standardized** ESLint config by `@react-native-community` (plus an associated `lint` shortcut in package.json), this also removes the now obsolete custom `standard` config and associated dependencies. In order to keep the line changes of this PR low, it does _not_ reformat any code yet (will be done in a follow-up PR) - I've _temporarily_ excluded `src/` and `index.js` from linting using `.eslintignore` (to avoid CI failure).

Tested and confirmed to work with ern `0.38.7` on

- Android Device (Xiaomi Mi A2)
- Android Emulator (Pixel 3a)
- iOS Device (iPhone 8)
- iOS Simulator (iPhone X)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,23 +74,5 @@
...
-      "it"
-    ],
-    "ignore": [
-      "flow-typed"
-    ],
-    "parser": "babel-eslint",
-    "plugins": [
-      "flowtype"
-    ]
-  }
```
Comment (cdaringe) [inline package.json]: standard in the upcoming 15 release has some exciting stuff, mainly around better 1st class TS support.  sad to see it go 👋

Comment (friederbluemle) [reply package.json]: Yeah, true, but fortunately ESLint also has some exciting stuff on the way that should help with better TypeScript support. Mainly the convergence with TSLint: https://medium.com/palantir/tslint-in-2019-1a144c2317a9 :) 👍

**Comment by cdaringe** [comment]
pulled @friederbluemle's branch and ran it.  works on my ios8 sim.  lgtm!

**Comment by friederbluemle** [comment]
Thanks a lot for testing this @cdaringe 👍 
We'll just do a few more tests before the merge, since I saw some unusual behavior on my Android device/emulator (could have been caused by local setup).

**Comment by friederbluemle** [comment]
@deepueg - Could you quickly verify that everything works as expected on the Android side? Specifically regarding the toolbar icons/buttons not showing up correctly, as I mentioned last week. I'm not sure if it was related to my local setup or something that changed in the update.

**Comment by deepueg** [comment]
Tried locally, works fine.

**Comment by friederbluemle** [comment]
Great, thanks @deepueg

**Pull request merged**